src/preprocessing.ipynb

```diff
@@ -37,7 +37,7 @@
     },
```

Comment:
<p>Move import to the top with the other imports</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geotrieu/transit-ai/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
This is in reference to the line
`import pandas as pd`